src/navbar/user-content.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { Gravatar } from 'navbar/gravatar'
+import { Email } from 'navbar/email'
+import { useStyles } from 'navbar/style'
+
+export const UserContent = () => <Grid item xs={6} sm={6} md={6} lg={6} xl={6}>
+  <Grid container direction={'row'} alignItems={'center'} justify={'flex-end'} className={useStyles().userContent}>
```

Comment:
same as with AppBar,, make the Grid thingy a wrapper and pass children inside... trust me people reading your code will  send you love letters!

Reply:
Done